app/actions/views/permalink.ts

```diff
@@ -15,31 +15,37 @@ export let showingPermalink = false;
 
 export function showPermalink(intl: typeof intlShape, teamName: string, postId: string, openAsPermalink = true) {
     return async (dispatch: DispatchFunc) => {
-        const loadTeam = await dispatch(loadChannelsByTeamName(teamName, permalinkBadTeam.bind(null, intl)));
-
-        if (!loadTeam.error) {
-            Keyboard.dismiss();
-            dispatch(selectFocusedPostId(postId));
-
-            if (!showingPermalink) {
-                const screen = 'Permalink';
-                const passProps = {
-                    isPermalink: openAsPermalink,
-                    onClose: () => {
-                        dispatch(closePermalink());
-                    },
-                };
-
-                const options = {
-                    layout: {
-                        componentBackgroundColor: changeOpacity('#000', 0.2),
-                    },
-                };
-
-                showingPermalink = true;
-                showModalOverCurrentContext(screen, passProps, options);
+        // For the special, "_redirect" path, just use the currently selected team.
```

Comment:
This is just to avoid the error of trying to load an invalid team.

Reply:
Hmm, I thought I had purged the vestigial parts of my initial investigation, but it looks like you're right: this isn't needed. Early on, I was encountering issues with what I thought was a proactive metadata fetch from orthogonal code, but I can no longer reproduce same. I've reverted all the corresponding changes.